diff of execution of:
echo 'echo a > out; cat infile > out >> out; ls -l --time-style=+' | tr ';' '\n' | [shell]
--- expected	2024-08-27 20:20:58.801947813 +0200
+++ found	2024-08-27 20:20:58.819948015 +0200
@@ -1,9 +1,9 @@
 total 8
 -rw-rw-r-- 1 bruno bruno 34  infile
--rw-rw-r-- 1 bruno bruno 34  out
+-rw-r--r-- 1 bruno bruno 34  out
 -r--r--r-- 1 bruno bruno  0  readonly
 
 ======== stderr =======
 
 ======== return =======
 return value: 0
